catalog/ui/search-ui/simple/src/main/resources/SearchPage.jsp

```diff
@@ -475,5 +475,29 @@
 
  	<script type="text/javascript" src="js/Search-min.js?bust=${timestamp}"></script>
 
+    <!-- Usage Modal -->
+    <!-- Only show if the modal is enabled -->
+    <% if(props.getSystemUsageEnabled() == true) { %>
+    <!-- Do not allow users to click outside of the modal to dismiss it, the OK button must be pressed -->
+    <script>
+        jQuery(function () {
+            $('#usageModal').modal({
+                backdrop: 'static',
```

Comment:
Prevents users from clicking outside of the modal or pressing escape to dismiss it. 
Also adds darkens the page behind the modal to bring focus onto it.